src/hotspot/share/runtime/threads.cpp

```diff
@@ -450,8 +455,8 @@ jint Threads::create_vm(JavaVMInitArgs* args, bool* canTryAgain) {
   jint parse_result = Arguments::parse(args);
   if (parse_result != JNI_OK) return parse_result;
 
-  // Initialize NMT right after argument parsing to keep the pre-NMT-init window small.
-  MemTracker::initialize();
+  // Verify NMT flag(s) and log the initial state to the ouput
```

Comment:
Nit: So far in this PR you have used the terms "NMT property", "NMT setting", "NMT arguments", and now "NMT flag(s)". Please pick one (not "property") and use consistently.

Reply:
Typo: ouput

Reply:
Fixed.